changelog.rst

```diff
@@ -0,0 +1,266 @@
+=========
+Changelog
+=========
+
+The **garnett** package follows `semantic versioning <https://semver.org/>`_.
+
+Version 0.5
+===========
+
+[0.5.0] -- 2019-0x-xx
+---------------------
+Added
++++++
+  - Rowan is now a dependency.
```

Comment:
Since some people might not be familiar with the package, I would try to state this more explicitly and add a link:

> Add the [rowan](https://rowan.readthedocs.io/en/latest/) package as a dependency.